src/screens/GroupInfo/hooks/useGroupInfo.js

```diff
@@ -334,8 +338,9 @@ const useGroupInfo = () => {
   const handleMessageAnonymously = async () => {
     try {
       setIsLoadingInitChat(true);
-      await handleAnonymousMessage(selectedUser);
+      await handleAnonymousMessage(selectedUser, channelId, 'chat');
     } catch (e) {
+      console.log(e);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Error handling in `handleMessageAnonymously` function has been improved by adding a console log statement. However, it would be better to handle the error in a more user-friendly way rather than just logging it to the console. Consider showing an error message to the user or retrying the operation depending on the nature of the error.
```diff
- console.log(e);
+ // Show error message to the user or retry operation
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->